build/repositories.bzl

```diff
@@ -41,6 +41,16 @@ def github_cli_repositories():
             build_file_content = _SRCS_BUILD_FILE_CONTENT,
         )
 
+def kong_github_repositories():
+    maybe(
+        github_release,
+        name = "kong_admin_gui",
+        repo = "kong/kong-manager-oss",
```

Comment:
It seems `kong/kong-manager-oss` is already private, This PR is still can not be built

Reply:
Yes the open source request is still in progress, we won't merge this PR until we get approval